### PR TITLE
Update proxy core

### PR DESCRIPTION
**Pull request by georgifarashev**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -27,11 +27,11 @@
 
 [[constraint]]
   name = "github.com/kubernetes-incubator/service-catalog"
-  version = "=0.1.27"
+  version = "=0.1.36"
 
 [[constraint]]
   name = "github.com/Peripli/service-broker-proxy"
-  branch = "master"
+  version = "0.1.1"
```
Comment (dpanayotov) [inline Gopkg.toml]: v0.1.1

Comment (georgifarashev) [reply Gopkg.toml]: Why? I see that even though the tag/release is v0.1.1 it is ok to use only 0.1.1. We do it like this for all other dependencies.

Comment (georgifarashev) [reply Gopkg.toml]: Why? I see that even though the tag/release is v0.1.1 it is ok to use only 0.1.1. We do it like this for all other dependencies.

Comment (dpanayotov) [reply Gopkg.toml]: I've seen both.. don't know which is the preferred one

**Comment by dpanayotov** [comment]
Can you also update the application.yml and values.yml with the new values -  https://github.com/Peripli/service-broker-proxy-cf/pull/20/files

**Pull request merged**